### PR TITLE
translations: update ruRU

### PR DESCRIPTION
**Pull request by shikulja**

### CODE REVIEW

File: env/translations_ruRU.lua [hunk 1]
```diff
@@ -297,7 +297,7 @@ pfUI_translation["ruRU"] = {
   ["Experience Color"] = "Цвет опыта",
   ["Export"] = "Экспорт",
   ["Face"] = "Лицо",
-  ["Fade To Custom Color"] = nil,
+  ["Fade To Custom Color"] = "Пользовательский цвет затухания",
```
Comment (shikulja) [inline env/translations_ruRU.lua:300]: ["Enable Custom Colors"] = "Включить пользовательские цвета",
  ["Enable Chat Fade"] = "Включить затухание чата",
Original text (English), slightly different stylistically.

**Comment by shikulja** [comment]
I checked everything in the game. All ok

**Comment by shikulja** [comment]
I checked everything in the game. All ok

**Comment by shagu** [comment]
Thanks :+1:

**Pull request merged**